src/main/java/com/tripply/Auth/controller/AuthController.java

```diff
@@ -50,4 +50,18 @@ public ResponseEntity<ResponseModel<String>> logout(HttpServletRequest request)
         log.info("Endpoint: logout response");
         return ResponseEntity.ok(response);
     }
+
+    @PostMapping(value = "/forgot-password", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<ResponseModel<String>> forgotPassword(@Valid @RequestBody String email){
+        log.info("Endpoint : /forgot-password email : {}" , email);
+        ResponseModel<String> response = authService.forgotPassword(email);
+        return ResponseEntity.ok(response);
```

Comment:
Start logs are there, Please add end logs also for controller.

Reply:
added end logs also.